example/grafana-dashboard.json

```diff
@@ -0,0 +1,264 @@
+{
```

Comment:
I'm not sure if this works. I started the docker-compose, logged in as admin/admin, and didn't see any dashboard configured.

Is there an extra step I need?

Reply:
yeah you need to import the dashboard once grafana starts